complex/ft_test.c

```diff
@@ -49,7 +49,7 @@ static int ft_init_xcontrol(struct ft_xcontrol *ctrl)
 	memset(ctrl, 0, sizeof *ctrl);
 	ctrl->credits = FT_DEFAULT_CREDITS;
 	ctrl->max_credits =  FT_DEFAULT_CREDITS;
-	ft_rx.comp_wait = FI_WAIT_NONE;
+	ctrl->comp_wait = test_info.cq_wait_obj;
```

Comment:
Somewhat similar, if FI_WAIT_NONE is *not* specified, then the code should call fi_cq_sread() instead of polling on fi_cq_read() in ft_comp_rx/tx. This way, we're at least testing the wait object code path.

For FI_WAIT_FD, we may want to go further an actually get the fd and call select/poll.  Though that's more complexity.  For now, I would just add calling sread.